daemon/api.go

```diff
@@ -882,3 +890,23 @@ func deleteCapability(c *Command, r *http.Request) Response {
 		return InternalError("can't remove capability %q: %v", name, err)
 	}
 }
+
+func doAssert(c *Command, r *http.Request) Response {
+	b, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		return BadRequest("reading assert request body gave %v", err)
+	}
+	a, err := asserts.Decode(b)
+	if err != nil {
+		return BadRequest("can't decode request body into an assertion: %v", err)
+	}
+	if err := c.d.asserts.Add(a); err != nil {
+		// TODO: have a specific error to be able to return  409 for not newer revision?
+		return BadRequest("assert failed: %v", err)
+	}
+	// TODO: what more info do we want to return on success?
```

Comment:
If each assertion has a URL and we've just created one perhaps the result should be 201 and along with an URL of the assertion. Then again I don't know how much http-friendly do we need to make the protocol. @chipaca ?